core/keyserver/keyserver.go

```diff
@@ -39,6 +39,8 @@ const (
 	defaultPageSize = 16
 	// Maximum allowed requested page size to prevent DOS.
 	maxPageSize = 16
+  // If no epoch is provided default to epoch 1.
```

Comment:
This doesn't look like the right indentation. Maybe it's only shown this way in github.